easybuild/easyconfigs/a/AlphaPulldown/AlphaPulldown-2.0.0-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,119 @@
+# created by Denis Kristak (Inuits)
+# update: Thomas Hoffmann, EMBL
+easyblock = 'PythonBundle'
+
+name = 'AlphaPulldown'
+version = '2.0.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/KosinskiLab/AlphaPulldown'
+description = """AlphaPulldown is a Python package that streamlines protein-protein
+interaction screens and high-throughput modelling of higher-order oligomers using AlphaFold-Multimer"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [
+    ('poetry', '1.5.1'),
+]
+
+dependencies = [
+    ('CUDA', '12.1.1', '', SYSTEM),
+    ('Python', '3.11.3'),
+    ('OpenMM', '8.0.0', versionsuffix),
+    ('Kalign', '3.4.0'),
+    ('PyYAML', '6.0'),
+    ('jax', '0.4.25', versionsuffix),  # also provides absl-py
+    ('Biopython', '1.83'),
+    ('h5py', '3.9.0'),
+    ('IPython', '8.14.0'),
+    ('JupyterLab', '4.0.5'),
```

Comment:
JupyterLab needs to be last.
jupyter-events from jupyter-server (which is a dep in JupyterLab) has a dependency for jsonschema >= 4.18.0 but everything else in this dep list uses 4.17.3.

Reply:
@akesandgren thanks. I moved it.